src/char/int_guild.c

```diff
@@ -1366,6 +1368,7 @@ int mapif_parse_GuildChangeMemberInfoShort(int fd, int guild_id, int account_id,
 		g->member[i].online = online;
 		g->member[i].lv = lv;
 		g->member[i].class = class;
+		g->member[i].last_login = (int)time(NULL);
```

Comment:
last_login field in unsigned int, why you cast here to signed int?
probably replace to ``(uint32)time(NULL)``

Reply:
I see, when I did the initial implementation I was using int, but then I went to look at the packet and noticed it was an unsigned int so I switched to uint32 and I ended up forgetting to change part of the time.

Reply:
Done.